src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,66 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        checkLogin(user);
+        checkPassword(user);
+        checkAge(user);
+        addOriginUser(user);
+        return user;
+    }
+
+    private static void checkLogin(User user) {
+        String exceptionMessage = "User login length should be at least 6 characters.";
```

Comment:
there is no need to declare a separate variable for the exception message - you can inline it